pwnlib/abi.py

```diff
@@ -123,5 +154,26 @@ class SigreturnABI(SyscallABI):
 linux_arm_sigreturn = SigreturnABI('sp', ['r7'], 4, 0)
 linux_aarch64_sigreturn = SigreturnABI('sp', ['x8'], 16, 0)
 
+freebsd_i386 = sysv_i386
+freebsd_amd64 = sysv_amd64
+freebsd_arm = sysv_arm
+freebsd_aarch64 = sysv_aarch64
+freebsd_mips = sysv_mips
+freebsd_ppc = sysv_ppc
+freebsd_ppc64 = sysv_ppc64
+
+freebsd_i386_syscall   = SyscallABI('esp', ['eax'], 4, 0)
+freebsd_amd64_syscall  = SyscallABI('rsp', ['rax','rdi','rsi','rdx','rcx','r8','r9'], 8, 0)
+freebsd_arm_syscall    = SyscallABI('sp', ['r7', 'r0', 'r1', 'r2', 'r3'], 8, 0)
+freebsd_aarch64_syscall = SyscallABI('sp', ['x8', 'x0', 'x1', 'x2', 'x3'], 16, 0)
+freebsd_mips_syscall  = SyscallABI('$sp', ['$v0','$a0','$a1','$a2','$a3'], 4, 0)
+freebsd_ppc_syscall = SyscallABI('sp', ['r0', 'r3', 'r4', 'r5', 'r6', 'r7', 'r8', 'r9', 'r10'], 4, 0)
+freebsd_ppc64_syscall = SyscallABI('sp', ['r0', 'r3', 'r4', 'r5', 'r6', 'r7', 'r8', 'r9', 'r10'], 8, 0)
+
+freebsd_i386_sigreturn = SigreturnABI('esp', ['eax'], 4, 0)
```

Comment:
Shouldn't `*_sigreturn` also be unified using `sysv_...`?

Reply:
Fixed